commands/dht.go

```diff
@@ -27,9 +28,72 @@ var dhtCmd = &cmds.Command{
 	},
 
 	Subcommands: map[string]*cmds.Command{
+		"query":     queryDhtCmd,
 		"findprovs": findProvidersDhtCmd,
 	},
 }
+var queryDhtCmd = &cmds.Command{
+	Helptext: cmdkit.HelpText{
+		Tagline:          "Find the closest Peer IDs to a given Peer ID by querying the DHT.",
+		ShortDescription: "Outputs a list of newline-delimited Peer IDs.",
+	},
+
+	Arguments: []cmdkit.Argument{
+		cmdkit.StringArg("peerID", true, true, "The peerID to run the query against."),
```

Comment:
Please set the third argument `false`.  go-ipfs defaults all of these to variadic even though the command body only every uses the first request argument which seems inconsistent.